modules/site-health/admin-site-health.php

```diff
@@ -108,6 +108,80 @@ function ( &$value, $key ) {
 		return implode( ' | ', $array );
 	}
 
+	/**
+	 * Transforms the option value to readable human sentence.
+	 *
+	 * @since 3.3
+	 *
+	 * @param string                $key option name.
+	 * @param string|array|bool|int $value option value.
+	 * @return mixed Option value.
+	 */
+	public function format_value( $key, $value ) {
+		switch ( $key ) {
+			case 'browser':
+				if ( ! $value ) {
+					$value = '0: ' . esc_html__( 'Browser prefered language deactivated', 'polylang' );
+					break;
+				}
+				$value = '1: ' . esc_html__( 'Browser prefered language activated', 'polylang' );
```

Comment:
```suggestion
					$value = '0: ' . esc_html__( 'Browser preferred language detection deactivated', 'polylang' );
					break;
				}
				$value = '1: ' . esc_html__( 'Browser preferred language detection activated', 'polylang' );
```

Reply:
I wonder if it would not be better to reuse the same string as in the Settings ? `Detect browser language activated/deactivated`

Reply:
As you want